scripts/gen-tx-gov.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+if test "$#" -ne 2
+then
+   echo "error: 2 parameters are expected (patch version and upgrade height number)"
+   echo "example : ./gen-tx-gov.sh v0.16.1 1336350"
+    exit
+fi
+
+if [[ ! "$1" =~ v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3} ]]; 
+then 
+   echo "error: a semantic tag is expected as first parameter"
+   echo  "example v0.16.1"
+   exit
+fi
+
+re='^[0-9]+$'
+if ! [[ "$2" =~ $re ]]
+then
+   echo "error: an integer is expected as second parameter" >&2
+   exit
+fi
+
+PATCH="$1"
+MINOR=$(echo $PATCH | sed 's/..$//g')
+HEIGHT_NUMBER="$2"
+
+# LINUX AMD64
+curl --silent -LJO "https://github.com/axelarnetwork/axelar-core/releases/download/${PATCH}/axelard-linux-amd64-${PATCH}.zip.sha256"
+LX_AMD64_HASH=$(cat axelard-linux-amd64-${PATCH}.zip.sha256)
+rm axelard-linux-amd64-${PATCH}.zip.sha256
+
+# LINUX ARM64
+curl --silent -LJO "https://github.com/axelarnetwork/axelar-core/releases/download/${PATCH}/axelard-linux-arm64-${PATCH}.zip.sha256"
+LX_ARM64_HASH=$(cat axelard-linux-arm64-${PATCH}.zip.sha256)
+rm axelard-linux-arm64-${PATCH}.zip.sha256
+
+# DARWIN ARM64
+curl --silent -LJO "https://github.com/axelarnetwork/axelar-core/releases/download/${PATCH}/axelard-darwin-arm64-${PATCH}.zip.sha256"
+DW_ARM64_HASH=$(cat axelard-darwin-arm64-${PATCH}.zip.sha256)
+rm axelard-darwin-arm64-${PATCH}.zip.sha256
+
+# DARWIN AMD64
+curl --silent -LJO "https://github.com/axelarnetwork/axelar-core/releases/download/${PATCH}/axelard-darwin-amd64-${PATCH}.zip.sha256"
+DW_AMD64_HASH=$(cat axelard-darwin-amd64-${PATCH}.zip.sha256)
+rm axelard-darwin-amd64-${PATCH}.zip.sha256
+
+cat <<EOF > txgov.txt | sed -e "s/LX_AMD64_HASH/$LX_AMD64_HASH/; s/LX_ARM64_HASH/$LX_ARM64_HASH/; s/DW_AMD64_HASH/$DW_AMD64_HASH/; s/DW_ARM64_HASH/$DW_ARM64_HASH/"
```

Comment:
can you add a comment here to explain what this does?

Reply:
sure.